geometry_msgs/PoseStamped pose
sensor_msgs/Image image
geometry_msgs/Transform surface_transform
